.circleci/config.yml

```diff
@@ -63,52 +64,10 @@ jobs:
                python -c "import mne; mne.sys_info()"
 
         # Run tests
-
-        - run:
-            name: test ds000246
-            command: |
-               export DATASET=ds000246
-               make fetch
-               export BIDS_ROOT=$DATA_DIR/ds000246
-               export MNE_BIDS_STUDY_CONFIG=config_ds000246
-               make sensor
-
-        - run:
-            name: test ds000248
-            command: |
-               export DATASET=ds000248
-               make fetch
-               export BIDS_ROOT=$DATA_DIR/ds000248
-               export MNE_BIDS_STUDY_CONFIG=config_ds000248
-               make sensor
-
-        - run:
-            name: test ds001810
-            command: |
-               export DATASET=ds001810
-               make fetch
-               export BIDS_ROOT=$DATA_DIR/ds001810
-               export MNE_BIDS_STUDY_CONFIG=config_ds001810
-               make sensor
-
-        - run:
-            name: test matchingpennies
-            command: |
-               export DATASET=eeg_matchingpennies
-               make fetch
-               export BIDS_ROOT=$DATA_DIR/eeg_matchingpennies
-               export MNE_BIDS_STUDY_CONFIG=config_matchingpennies
-               make sensor
-
         - run:
-            name: test somato data
+            name: run test suite
             command: |
-               export DATASET=somato
-               make fetch
-               export BIDS_ROOT=$DATA_DIR/somato
-               export MNE_BIDS_STUDY_CONFIG=config_somato
-               make sensor
-               make source
+               python tests/run_tests.py
```

Comment:
I would not do it this way. It's nice to see in circle a run per dataset. You see which dataset takes how long etc.
So the run_tests.py should run for one dataset at a time. Not for all datasets

my 2c

Reply:
so I would expect:

`python tests/run_tests.py ds000246 `

Reply:
sounds reasonable! Please see my adjustments

Reply:
> You see which dataset takes how long etc.

can't we use pytest for that instead of reinventing the wheel?

Reply:
> can't we use pytest for that instead of reinventing the wheel?

yes, but when you inspect the circleCI error log, it's much nicer to have a single green or red button per dataset --> it means that the logs that can fail or succeed are smaller to inspect.